app/views/devise/passwords/edit.html.slim

```diff
@@ -0,0 +1,24 @@
+h2 Change your password
+
+= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
+= devise_error_messages!
+= f.hidden_field :reset_password_token
+
+  .field
+    = f.label :password, "New password" 
+    br
+      = if @minimum_password_length
```

Comment:
This should be a hyphen - as far as I know. Because you aren't outputting the code onto the page